From a1c88691d4d9b0877da6e902d787d737262dd33d Mon Sep 17 00:00:00 2001
From: Rick Harris <rconradharris@gmail.com>
Date: Thu, 10 Nov 2011 23:33:45 +0000
Subject: [PATCH] Using Keystone's new port number 35357.

Fixes bug 888753

(cherry picked from commit 94dcf3acd0b3ef7adcb61b90bf4bdcc733cf61ac)

Change-Id: I22894a31ee54edd6d00ac3458ea4a2f8842aa4a0
---
 doc/source/authentication.rst |    2 +-
 etc/glance-api.conf           |    2 +-
 etc/glance-registry.conf      |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/source/authentication.rst b/doc/source/authentication.rst
index 8f3acf4..2d3810d 100644
--- a/doc/source/authentication.rst
+++ b/doc/source/authentication.rst
@@ -124,7 +124,7 @@ pieces of middleware exist.  Here is an example for ``authtoken``::
   service_host = 127.0.0.1
   service_port = 5000
   auth_host = 127.0.0.1
-  auth_port = 5001
+  auth_port = 35357
   auth_protocol = http
   auth_uri = http://127.0.0.1:5000/
   admin_token = 999888777666
diff --git a/etc/glance-api.conf b/etc/glance-api.conf
index f3ca0da..5e6956d 100644
--- a/etc/glance-api.conf
+++ b/etc/glance-api.conf
@@ -165,7 +165,7 @@ service_protocol = http
 service_host = 127.0.0.1
 service_port = 5000
 auth_host = 127.0.0.1
-auth_port = 5001
+auth_port = 35357
 auth_protocol = http
 auth_uri = http://127.0.0.1:5000/
 admin_token = 999888777666
diff --git a/etc/glance-registry.conf b/etc/glance-registry.conf
index c81e1a4..5f21702 100644
--- a/etc/glance-registry.conf
+++ b/etc/glance-registry.conf
@@ -58,7 +58,7 @@ service_protocol = http
 service_host = 127.0.0.1
 service_port = 5000
 auth_host = 127.0.0.1
-auth_port = 5001
+auth_port = 35357
 auth_protocol = http
 auth_uri = http://127.0.0.1:5000/
 admin_token = 999888777666
-- 
1.7.6.5

